<launch>
	
<!--
	sel=0  : many files
	sel=1  : 1 file
-->

<!-- -->	
	
	
	<include file="$(find kinect_driver)/launch/kinect_driver.launch" />
	<!--param name="kinect/driver/frame_rate" value="5.0" type="double" /-->
	
	<!--include file="$(find openni_camera)/launch/openni_node.launch" /-->
	
	
		
	<node pkg="raposang_icp" name="world_node" type="world_node">
		<param name="sel" value="1" type="int" />
		
		<param name="ply_filepath" value="/home/pedrovieira/Desktop/results/map_many_files/" type="str" />
		<param name="ply_filename" value="/home/pedrovieira/Desktop/results/cloud_world.ply" type="str" />
			
	</node>
	
	
</launch>
